ashwin_vidiyala/Rails/app/views/students/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <h2><%= @dojo.branch %>'s <%= @student.first_name %> <%= @student.last_name %></h2>
+    <p>Email: <%= @student.email %></p>
+
+    <h2>Cohort</h2>
+    <table>
+      <tr>
+        <th>First Name</th>
+        <th>Last Name</th>
+        <th>Email</th>
+      </tr>
+      <% @students.each do |student| %>
+        <% if student != @student %>
```

Comment:
Not lazy enough ;) Try to implement a method that does this for you.

Reply:
Haha yes sir. I have that method working now :)